style.css

```diff
@@ -574,8 +635,243 @@ button:active {
   background-color: #0d151e;
   margin-top: 1px;
   margin-bottom: 10px;
+  margin-left: 42%;
 }
 
 .desktop-nav {
   display: none;
 }
+
+@media screen and (min-width: 996px) {
+  .top-bar-left {
+    height: 72px;
+    padding-left: 154px;
+    padding-bottom: 26px;
+    padding-top: 26px;
+  }
+
+  .my-name {
+    display: block;
+    flex-wrap: nowrap;
+    font-size: 72px;
+    letter-spacing: 0.37px;
+    width: 566px;
+    height: 124px;
+    line-height: 81px;
+    font-weight: normal;
+    padding-top: 192px;
+    padding-left: 143px;
+  }
+
+  .desktop-nav {
+    display: flex;
+    justify-content: space-between;
+    margin-right: 151px;
+    width: 320px;
+    height: 124px;
+  }
+
+  .top-bar-links {
+    font-weight: bold;
+    margin-left: auto;
+    background-color: #3c3a39;
+    padding-right: 24px;
+    padding-top: 24px;
+    height: 72px;
+  }
+
+  .desktop-link {
+    color: #fff;
+    font-family: "Inter", sans-serif;
+    font-weight: 600;
+    font-size: 15px;
+    align-items: center;
+  }
+
+  .top-bar-icon {
+    display: none;
+  }
+
+  .subtext {
+    font-size: 32px;
+    line-height: 44px;
+    padding-top: 148px;
+    padding-left: 143px;
+    width: 566px;
+  }
+
+  .text {
+    font-size: 20px;
+    text-shadow: 1px 1px #1c1a19;
+    margin-top: 12px;
+    padding-right: 24px;
+    padding-left: 143px;
+    font-weight: 400;
+    width: 720px;
+    display: flex;
+  }
+
+  .social-icons {
+    position: absolute;
+    top: 330px;
+    padding-left: 36px;
+    justify-content: flex-start;
+    flex-direction: column;
+  }
+
+  .background-desktop {
+    background: url("assets/background_desktop.png");
+    background-repeat: no-repeat;
+    background-color: #1c1a19;
+    background-size: 45%;
+    background-position: right bottom;
+    gap: 12px;
+  }
+
+  .line {
+    margin-top: -50px;
+    margin-left: 350px;
+  }
+
+  .text2 {
+    margin-top: -460px;
+    margin-left: 700px;
+  }
+
+  .text3 {
+    margin-left: 700px;
+  }
+
+  .link-tag {
+    margin-left: 700px;
+  }
+
+  .space {
+    padding-top: 200px;
+  }
+
+  .btn2 {
+    margin-left: 700px;
+    max-height: 48px;
+  }
+
+  .container-section1 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .container-section2 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .backgroundabout-me {
+    background: url("assets/background_desktop.svg");
+    background-repeat: no-repeat;
+    background-position: right;
+  }
```

Comment:
- [ ] In the mobile view, it's the paragraph in this section then one causing the page to overflow. Let's try to fix it aswell.

![image](https://user-images.githubusercontent.com/79114233/140407603-bf23a958-9565-447c-a2da-4e1f260c1296.png)

Reply:
Fixed by setting a width of 327px to element